vege/style.css

```diff
@@ -34,6 +35,157 @@ body {
 }
 
 
+
+
 /* ------------------------------- */
 /* ----- DOLE BUDE TVOJE CSS ----- */
 /* ------------------------------- */
+.container {
+    flex-direction: column;
+}
+h1 {
+    font-weight: 800;
+    font-size: 1.75rem;
+    padding: 10rem 0.5rem 0.5rem 0.5rem;
+    margin: 0;
+}
```

Comment:
V návodu je taky stín za písmem, kdybys chtěla ten, bude to text-shadow: 0 2px 2px rgba(0, 0, 0, 0.3);